airgun/views/host_new.py

```diff
@@ -406,12 +407,12 @@ class errata(PF5Tab):
 
             select_all = Checkbox(locator='.//div[@id="selection-checkbox"]/div/label')
             searchbar = SearchInput(locator='.//input[contains(@class, "pf-v5-c-text-input")]')
-            type_filter = Select(locator='.//div[@aria-label="select Type container"]/div')
-            severity_filter = Select(locator='.//div[@aria-label="select Severity container"]/div')
+            type_filter = PF5OUIASelect(component_id='select Type')
+            severity_filter = PF5OUIASelect(component_id='select Severity')
```

Comment:
@pnovotny non blocking note for the future, `component_id` does not need to be specified, you can just directly pass the id to the definition. It looks cleaner imo.

Reply:
The explicit `component_id` parameter name is intentional and kinda a habit. 
Sometimes - not this particular case - it's not clear by the import aliases, if the widget is a _generic_ one / _PF_ one or _PF OUIA_ one. And each of them is accepting different init parameters. This helps (me) to distinguish them.

Reply:
I am firmly on team "explicit can be good, sometimes" when it comes to airgun. I find that I have to fix less issues with stuff where this is spelled out than when it's not, and often it's easier to bugfix/grok code that is more this way.